build/vagrant/infra/policy/s3_upload_policy.json

```diff
@@ -0,0 +1,29 @@
+{
+    "Statement": [
+        {
+            "Action": [
+                "s3:putObject"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:s3:::${artifact_location}/*",
+            "Sid": "AllowS3Upload"
+        },
+        {
+            "Action": [
+                "ssm:GetParameter"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:ssm:us-east-1:701759196663:parameter${ssm_vagrantcloud_token}",
+            "Sid": "AllowSsmRetrieve"
+        },
+        {
+            "Action": [
+                "kms:Decrypt"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:kms:*:*:key/f2cfe6f1-ea79-4f50-b4cf-53d9f25fdc8d",
```

Comment:
Shouldn't this not point to a specific kms key, but be parameterized?

Reply:
Fixed :)